### PR TITLE
Cleans up LHC of homepage, particularly on mobile.

### PR DESCRIPTION
**Pull request by gregmarra**
RFC, since this changes some nav.

## Description

1. Shorten elements of LHC, such as site description
2. Remove duplicated Events / Teams buttons from LHC, since they are in top nav
3. Remove duplicated Search box from LHC, since it is in top nav
4. Move Avatars button to /teams
5. On small screens, move advanced search buttons into navbar more menu
6. Change Search hint text to have an emoji, because twitter-typeahead component makes adding a bootstrap input-group glyphicon submit button hard
7.  Rearrange nav menu, putting GameDay before Insights (more popular) and moving Blog to more menu

Changes are duplicated in jinja1 and jinja2 base.html files.

## Motivation and Context
More stuff above the fold on mobile

## How Has This Been Tested?
Tested on local machine at each screen width size to confirm all nav elements available.

## Screenshots (if appropriate):

Before
screenshot 2019-02-27 11 47 32
screenshot 2019-02-27 11 47 37

After
screenshot 2019-02-27 11 47 12
screenshot 2019-02-27 11 47 18

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

**Comment by fangeugene** [comment]
How does it look on desktop?

**Comment by gregmarra** [comment]
Before
screenshot 2019-02-27 13 00 55

After
screenshot 2019-02-27 13 00 41

**Comment by fangeugene** [comment]
I like it. Where do avatars go, though?

File: templates/base.html [hunk 1]
```diff
@@ -81,14 +81,20 @@
...
         <li class="visible-xs {% block insights_active_collapsed %}{% endblock %}"><a href="/insights"><span><span class="glyphicon glyphicon-align-left"></span> Insights</span></a></li>
         <li class="visible-xs"><a href="/gameday"><span><span class="glyphicon glyphicon-facetime-video"></span> GameDay</span></a></li>
-        <li class="visible-xs"><a href="//blog.thebluealliance.com"><span><span class="glyphicon glyphicon-pencil"></span> Blog</span></a></li>
+        <li class="visible-xs visible-sm"><a href="//blog.thebluealliance.com"><span><span class="glyphicon glyphicon-pencil"></span> Blog</span></a></li>
         <li class="visible-xs visible-sm"><a href="/account"><span class="glyphicon glyphicon-cog"></span> Account</a></li>
       </ul>
 
       <form class="navbar-form tba-navbar-form" action="/search" >
-        <input type="text" class="form-control typeahead" placeholder="Search teams and events" name="q"/>
+        <input type="text" class="form-control typeahead" placeholder="🔍 Search teams and events" name="q"/>
```
Comment (ZachOrr) [inline templates/base.html]: 🔍

**Comment by gregmarra** [comment]
Avatars are now a button on /teams

**Comment by fangeugene** [comment]
Are we good with this? I know people in Slack had some opinions on keeping more stuff on Desktop.

**Comment by gregmarra** [comment]
I think it's more focused, better encourages app downloads, and teaches
people to use the top nav bar better. I'd prefer to commit this and debate
what to add back to Desktop, just to improve the mobile experience.

**Pull request merged**